web_notify/static/src/js/widgets/notification.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('web_notify.Notification', function (require) {
+    "use strict";
+
+    var Notification = require('web.Notification');
+
+    Notification.include({
+        init: function () {
+            this._super.apply(this, arguments);
+            // Delete default classes
+            this.className = this.className.replace(' o_error', '');
+            var addClass = 'o_' + this.type;
+            if (this.type === 'success') {
```

Comment:
pls define a mapping to be included in this class (`this.icon_mapping[this.type]` is going to be easier to customize and makes the code more clean